boleto/templateCaixa.go

```diff
@@ -357,6 +357,16 @@ const boletoFormCaixa = `
                 <td colspan="6" rowspan="6">
                     <span class="title">Instruções de responsabilidade do BENEFICIÁRIO. Qualquer dúvida sobre este boleto contate o beneficiário.</span>
                     <p class="content" id="instructions">{{.View.Boleto.Title.Instructions }}</p>
+                    {{if .View.Boleto.Title.HasFees}}
```

Comment:
@brunoglm Acho que vale puxar a identação de `if`` pra trás só pra dar mas legibilidade... eu particularmente prefiro quando a instrução tbm está identada, como nós códigos normais. Apesar de não ter linter acho q ajuda na leitura:
```suggestion
              {{if .View.Boleto.Title.HasFees}}
                    <br/>
                    <p class="content">** VALORES EXPRESSOS EM REAIS **</p>
                    {{if .View.Boleto.Title.Fees.Interest.HasInterest}}
                        <p class="content">{{getInterestInstruction .View.Boleto.Title}}</p>
                    {{end}}
                    {{if .View.Boleto.Title.Fees.Fine.HasFine}}
                        <p class="content">{{getFineInstruction .View.Boleto.Title}}</p>
                    {{end}}
               {{end}}
```

Reply:
Done